<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/><meta name="exporter-version" content="Evernote Mac 7.14 (458244)"/><meta name="author" content="taylor.arhar@gmail.com"/><meta name="created" content="2018-10-01 18:32:54 +0000"/><meta name="source" content="desktop.mac"/><meta name="updated" content="2018-10-02 20:15:29 +0000"/><title>October 1, 2018</title></head><body><div>Sequencing Jason Young Lab plasmids:</div><div>- after adding the new pCDNA3.1 sequencing primer, reads were long enough and overlapping with backbone, allowing for assembly of plasmid maps for all 4 constructs</div><div>- see attached for details</div><div>- RESULTS: at the C-terminus,Â all constructs have a short linker (8 aa) before a myc tag, followed by a 5 aa linker and a 6X his tag</div><div><a href="October%201,%202018.html.resources/092718%20Young%20Lab%20Plasmid%20Sequencing.docx">092718 Young Lab Plasmid Sequencing.docx</a><br/></div><div><br/></div><div>Design primers for subcloning:</div><div>- GOAL: subclone chimeras/truncations from pCDNA3.1 into pMCSG7 plasmid, for future bacterial expression</div><div>- see attached for details</div><div>- workflow: PCR amplify the inserts from the pCDNA3.1 plasmids, while introducing relevant restriction sites and other features (TEV cleavage site and stop codon); then empty pMCSG7 and the PCR products can be digested and ligated</div><div>- plasmid maps were assembled for each of these constructs in the pMCSG7 vector, including annotation with the relevant primers</div><div><a href="October%201,%202018.html.resources/100118%20DJA2%20chimera%20primers.xlsx">100118 DJA2 chimera primers.xlsx</a><br/></div><div><br/></div></body></html>
